zebra/redistribute.c

```diff
@@ -616,7 +616,6 @@ void zebra_interface_vrf_update_del(struct interface *ifp, vrf_id_t new_vrf_id)
 		zsend_interface_update(ZEBRA_INTERFACE_DOWN, client, ifp);
 		client->ifdel_cnt++;
 		zsend_interface_delete(client, ifp);
-		zsend_interface_vrf_update(client, ifp, new_vrf_id);
```

Comment:
How it will be taken care at the client side.
Example pimd client: 
In the function, pim_zebra_interface_vrf_update() it will update the interface to new VRF. 
If zebra is not sending the vrf_update msg then how it will be taken care.